chia/full_node/subscriptions.py

```diff
@@ -92,7 +92,7 @@ def add_puzzle_subscriptions(self, peer_id: bytes32, puzzle_hashes: List[bytes32
         added: Set[bytes32] = set()
 
         def limit_reached() -> Set[bytes32]:
-            log.info(
+            log.warning(
```

Comment:
Per earlier discussion - let's revert this back to info

Reply:
It has been reverted ⏪